integreat_cms/cms/views/pages/page_tree_view.py

```diff
@@ -81,13 +84,46 @@ def get(self, request, *args, **kwargs):
                 "language_tree",
                 **{
                     "region_slug": region.slug,
-                }
+                },
             )
 
         if not request.user.has_perm("cms.change_page"):
-            messages.warning(
-                request, _("You don't have the permission to edit or create pages.")
-            )
+            access_granted_pages = Page.objects.filter(
+                Q(editors=request.user) | Q(publishers=request.user)
+            ).filter(region=request.region)
+            if request.user.organization:
+                access_granted_pages = access_granted_pages.union(
+                    Page.objects.filter(organization=request.user.organization)
+                )
+            if len(access_granted_pages) > 0:
+                messages.error(
```

Comment:
```suggestion
                messages.info(
```
I wouldn't consider this an error